easybuild/easyblocks/o/openmpi.py

```diff
@@ -124,11 +124,11 @@ def config_opt_used(key, enable_opt=False):
                 # auto-detect based on available OS packages
                 os_packages = EASYCONFIG_CONSTANTS['OS_PKG_IBVERBS_DEV'][0]
                 verbs = any(check_os_dependency(osdep) for osdep in os_packages)
-
-            if verbs:
-                self.cfg.update('configopts', '--with-verbs')
-            else:
-                self.cfg.update('configopts', '--without-verbs')
+            if LooseVersion(self.version) <= LooseVersion('5.0.0'):
```

Comment:
@robert-mijakovic Can you clarify this? If `--with-verbs` no longer valid in OpenMPI 5.0.0?

How should we explicitely disable/enable IB support then for OpenMPI >= 5.0.0?

Reply:
IB support in 5 comes only from UCX if I remember correctly??

Reply:
@akesandgren Yes, that is my impression too.

Reply:
OK, let's add a comment above this `if` with a reference to https://github.com/open-mpi/ompi/pull/6270, just so we're not surprised by this going forward.